crates/biome_cli/src/cli_options.rs

```diff
@@ -82,7 +82,7 @@ pub struct CliOptions {
 
 impl CliOptions {
     /// Computes the [ConfigurationBasePath] based on the options passed by the user
-    pub(crate) fn as_configuration_base_path(&self) -> ConfigurationBasePath {
+    pub(crate) fn to_configuration_base_path(&self) -> ConfigurationBasePath {
```

Comment:
Sorry for the distraction, just renamed this to follow Rust's naming conventions: https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv